README.md

```diff
@@ -524,30 +532,81 @@ classDiagram
         model_version: str
     }
 
+    class MapShapeTypes {
+        <<Enumeration>>
+        rectangular: str = 'rectangular'
+        non_rectangular: str = 'non_rectangular'
+    }
+
     class MapColorSchemeTypes {
         <<Enumeration>>
         full_color: str = 'full_color'
         monochrome: str = 'monochrome'
         grayscale: str = 'grayscale'
     }
 
-    class CogMetaData {
-        cog_id: str
-        system: str
-        system_version: str
-        multiple_maps: Optional[bool]
-        map_metadata: Optional[list[MapMetaData]]
+    MapMetaData ..> MapShapeTypes
+    MapMetaData ..> MapColorSchemeTypes
+    CogMetaData ..> MapMetaData
+
+
+```
+
+</details>
+
+### Document
+
+<details open>
+    <summary>document</summary>
+
+```mermaid
+classDiagram
+
+    class Document {
+        id: UUID
+        title: str
+        is_public: bool
+        xdd_id: Optional[str] = None
```

Comment:
@iross I don't think we want the CDR to be tightly coupled to XDD - suggest removing the xdd_id.